cmd/thanos/query.go

```diff
@@ -289,7 +307,8 @@ func runQuery(
 					}
 					fileSDCache.Update(update)
 					stores.Update(ctxUpdate)
-					dnsProvider.Resolve(ctxUpdate, append(fileSDCache.Addresses(), storeAddrs...))
+					dnsStoreProvider.Resolve(ctxUpdate, append(fileSDCache.Addresses(), storeAddrs...))
+					// rules apis do not support file service discovery as of now.
```

Comment:
Full sentence please (First letter uppercase) 

Our CI check did find  that? :thinking:

Reply:
whopsie

Reply:
fixed